tests/integration/screenshots_test.ts

```diff
@@ -0,0 +1,36 @@
+import { ChromeClient, FirefoxClient } from "../../mod.ts";
+const ScreenshotsFolder =
+  ((Deno.build.os == "windows") ? "ScreenshotsInteg" : "./ScreenshotsInteg");
+
+try {
+  Deno.removeSync(ScreenshotsFolder, { recursive: true });
+} catch (_e) {
+  //
+} finally {
+  Deno.mkdirSync(ScreenshotsFolder);
+}
```

Comment:
Can this be moved into each test case? reason being is that for the tutorials in the docs, it will be inside the tets case because we wouldnt expect a user to write tests for both browser, it also means we dont need the rmsync at the bottom of the file

These integration tests are supposed to represent the tests on drash.land

Reply:
Undertood, will do.